frontend/routes/app/app.component.tsx

```diff
@@ -83,7 +83,7 @@ export class App extends React.PureComponent<IProps, IState> {
 	});
 
 	public renderHeader = renderWhenTrue(() => (
-		<TopMenu onLogout={this.props.logout} onLogoClick={this.handleLogoClick} />
```

Comment:
Any reason why we're removing the logo click?

At the moment if you click on the logo it'll go to teamspace view. with this removed it no longer happens unless we're in the viewer.

Reply:
Both `logout` action and `handleLogoClick` were only moved from `app.component` to `topMenu.component`, it doesn't have any impact on functionality it's just code optimization.